### PR TITLE
[ Doc ] Add explains about Applications

### PR DESCRIPTION
**Pull request by jijoongmoon**
Modify Document of Training Exmaple (Transfer Learning)
. Applications/Training/README.md

Signed-off-by: jijoongmoon <jijoong.moon@samsung.com>

### CODE REVIEW

File: Applications/Training/README.md [hunk 1]
```diff
@@ -2,20 +2,85 @@
...
 In this toy example, fully connected layer is added to classify as below. The size of the feature is also same with previous example and the training set and test set are also same. ( 3 classes, 5 training set and 2 test set of each )
-Only fully connected layer is updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
-I wrote some code for the forward and backward propagation during fully connected layer training.
+Only fully connected layers are updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
 
-Sigmoid function is used for the activation. Square error loss function and gradient decent is used for the loss function and optimizer. Just 20 neuron is used for the hidden layer and iterate 300 times to make is more simple.
+Sigmoid function is used for the activation. Square error loss function and gradient decent is used for the loss function and optimizer. Two fully connected layers are added for the hidden layer and iterate 300 times to make is more simple.
```
Comment (kparichay) [inline Applications/Training/README.md:7]: Square error -> Mean Sqaured Error
gradient descent -> Stochastic Gradient Descent

Comment (kparichay) [inline Applications/Training/README.md:7]: `Sigmoid function is used for the activation.` -> lets improve this. Its quite vague without knowing which layer we are talking about. I thought it was the final activation but that's softmax.
either way, if we talk about activation, it must say where that activation is.

Comment (jijoongmoon) [reply Applications/Training/README.md:7]: fixed. thanks.

Comment (jijoongmoon) [reply Applications/Training/README.md:7]: fixed. thanks.

**Review by kparichay — APPROVED** [review]
LGTM

File: Applications/Training/README.md [hunk 2]
```diff
@@ -2,20 +2,85 @@
...
+```bash
+# Network Section : Network ignored line started with '#'
+[Network]                     # Network Configuration
+Type = NeuralNetwork          # Network type
+Learning_rate = 0.01          # Learning rate
+Epoch = 100                   # Epoch
+Optimizer = sgd               # Optimizer to apply gradient
+Cost = cross                  # Cost function. last layer uses softmax as an activation.
+                              # So, it is Cross-Entropy with softmax
+Model = "model.bin"           # Updated weights are stored in files named 'model.bin'
```
Comment (zhoonit) [inline Applications/Training/README.md:46]:
```suggestion
save_path = "model.bin"           # Updated weights are stored in files named 'model.bin'
```

File: Applications/Training/README.md [hunk 3]
```diff
@@ -2,20 +2,85 @@
...
+minibatch = 1                 # Mini batch size (used it throughout layers)
```
Comment (zhoonit) [inline Applications/Training/README.md:47]:
```suggestion
batch_size = 1                 # Mini batch size (used it throughout layers)
```

**Review by zhoonit — APPROVED** [review]
LGTM with minor comment

File: Applications/Training/README.md (hunk 3 above)
Comment (jijoongmoon) [reply Applications/Training/README.md]: fixed. thanks

File: Applications/Training/README.md (hunk 2 above)
Comment (jijoongmoon) [reply Applications/Training/README.md]: fixed. thanks
